gravity.sh

```diff
@@ -113,25 +116,32 @@ gravity_store_in_database() {
     gravity_Cleanup "error"
   fi
 
+  local tmpFile=$(mktemp -p "/tmp" --suffix=".gravity")
+  if [ "$table" == "whitelist" ] || [ "$table" == "blacklist" ] || [ "$table" == "regex" ]; then
+    # Apply format for white-, blacklist, and regex tables, prevent globbing
+    set -f
+    for domain in $(cat < "${source}"); do
+      echo "\"${domain}\",1,$(date --utc +'%s')," >> "${tmpFile}"
```

Comment:
Will this cause the `comment` column to have an empty string? The importer fills the missing columns with null, so the extra comma is not necessary.